src/components/markdown-renderer/replace-components/click-shield/click-shield.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import type { IconName } from '../../../common/fork-awesome/types'
+import { ShowIf } from '../../../common/show-if/show-if'
+import './click-shield.scss'
+import { ProxyImageFrame } from '../image/proxy-image-frame'
+import { Logger } from '../../../../utils/logger'
+import type { Property } from 'csstype'
+import type { PropsWithDataCypressId } from '../../../../utils/cypress-attribute'
+import { cypressId } from '../../../../utils/cypress-attribute'
+import { ForkAwesomeIcon } from '../../../common/fork-awesome/fork-awesome-icon'
+
+const log = new Logger('OneClickEmbedding')
+
+interface ClickShieldProps extends PropsWithDataCypressId {
+  onImageFetch?: () => Promise<string>
+  fallbackPreviewImageUrl?: string
+  hoverIcon?: IconName
+  hoverTextI18nKey?: string
+  targetDescription?: string
+  containerClassName?: string
+  fallbackBackgroundColor?: Property.BackgroundColor
+}
+
+/**
+ * Prevents loading of the children elements until the user unlocks the content by e.g. clicking.
+ *
+ * @param containerClassName Additional CSS classes for the complete component
```

Comment:
Why not call this attribute `additionalClasses` like in [other](https://github.com/hedgedoc/react-client/blob/0aae1f70d20c425335421981f76cd58fc520967e/src/components/common/user-avatar/user-avatar.tsx#L16) [places](https://github.com/hedgedoc/react-client/blob/a398660c18e89489bd08f2925a61f2a54c282d8a/src/components/common/modals/common-modal.tsx#L24)?

Reply:
Because that would tell you where the classes will be added.

Reply:
ok